khmer/kfile.py

```diff
@@ -157,7 +157,9 @@ def check_valid_file_exists(in_files):
     or non-existent.
     """
     for in_file in in_files:
-        if os.path.exists(in_file):
+        if in_file == '-' or in_file == '/dev/stdin':
```

Comment:
'/dev/stdin' passes `os.path.exists()` unless it truly doesn't exist in which case it would be a real error (and a screwed up system).